bin/faucet/src/client.rs

```diff
@@ -314,19 +297,17 @@ fn build_transaction_arguments(
     let tag = output_note.metadata().tag().inner();
     let aux = output_note.metadata().aux().inner();
 
-    let script = ProgramAst::parse(
-        &DISTRIBUTE_FUNGIBLE_ASSET_SCRIPT
-            .replace("{recipient}", &recipient)
-            .replace("{note_type}", &Felt::new(note_type as u64).to_string())
-            .replace("{aux}", &Felt::new(aux).to_string())
-            .replace("{tag}", &Felt::new(tag.into()).to_string())
-            .replace("{amount}", &Felt::new(asset.amount()).to_string()),
-    )
-    .expect("shipped MASM is well-formed");
+    let script = &DISTRIBUTE_FUNGIBLE_ASSET_SCRIPT
+        .replace("{recipient}", &recipient)
+        .replace("{note_type}", &Felt::new(note_type as u64).to_string())
+        .replace("{aux}", &Felt::new(aux).to_string())
+        .replace("{tag}", &Felt::new(tag.into()).to_string())
+        .replace("{amount}", &Felt::new(asset.amount()).to_string());
```

Comment:
This is missing the execution hint but I believe they are also being implemented on #452 so it should be fine

Reply:
Ah - nice!